PelotonEppSdk/Interfaces/ICreditCardUpdateRequest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using PelotonEppSdk.Models;
+
+namespace PelotonEppSdk.Interfaces
+{
+    public interface ICreditCardUpdateRequest
+    {
+        string OrderNumber { get; set; }
+        string CardOwner { get; set; }
+
+        CreditCardNumber CardNumber { get; set; }
```

Comment:
We are not going to allow updates to CardNumbers. A create credit card request will need to be sent when changing the card number.